pass/ProteusPass.cpp

```diff
@@ -695,25 +756,24 @@ struct ProteusJitPassImpl {
 
     FunctionType *JitEntryFnTy = nullptr;
 #if ENABLE_HIP
-    JitEntryFnTy =
-        FunctionType::get(Int32Ty,
-                          {ModuleUniqueId->getType(), VoidPtrTy,
-                           FatbinWrapper->getType(), Int64Ty, Int64Ty, Int32Ty,
-                           Int64Ty, Int32Ty, VoidPtrTy, Int64Ty, VoidPtrTy},
-                          /* isVarArg=*/false);
+    JitEntryFnTy = FunctionType::get(
+        Int32Ty,
+        {ModuleUniqueId->getType(), PtrTy, FatbinWrapper->getType(), Int64Ty,
+         Int64Ty, Int32Ty, Int64Ty, Int32Ty, PtrTy, Int64Ty, PtrTy},
+        /* isVarArg=*/false);
```

Comment:
I was confused by the naming convention: `JitEntryFnTy` maps to both jit_entry and jit_launch_kernel. Maybe over here rename variable to `JitLaunchKernelFnTy`?

Reply:
Yes, makes the code clearer. As per your comment above, it makes sense to build encapsulating methods that are smartly named